modules/checklist/assets/js/components/checklist-item.js

```diff
@@ -14,14 +16,23 @@ function CheckListItem( props ) {
 	return (
 		<>
 			<ListItemButton onClick={ handleExpandClick } className={ `e-checklist-item-button checklist-step-${ step.config.id }` }>
-				<ListItemIcon>{ step.is_completed ? <CheckedCircleIcon /> : <RadioButtonUncheckedIcon />
-				}</ListItemIcon>
+				<ListItemIcon>
+					<Checkbox
+						className={ `step-check-icon ${ isChecked ? 'checked' : 'unchecked' }` }
+						icon={ <RadioButtonUncheckedIcon /> }
+						checkedIcon={ <CircleCheckFilledIcon color="primary" /> }
+						edge="start"
+						checked={ isChecked }
+						tabIndex={ -1 }
+						inputProps={ { 'aria-labelledby': step.config.title } }
+					/>
```

Comment:
Nice!

Reply:
Thanks, I got the idea from some brilliant developer :)